### PR TITLE
Updated docs.

### PR DESCRIPTION
**Pull request by dend**

### CODE REVIEW

**Comment by jhutchings1** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[xml/FrameworksIndex/healthvault-1.65.20615.4-preview.xml](https://github.com/dend/healthvault-docs/blob/master/xml/FrameworksIndex/healthvault-1.65.20615.4-preview.xml) | :white_check_mark:Succeeded | |
[xml/FrameworksIndex/healthvault-1.65.20623.2-preview.xml](https://github.com/dend/healthvault-docs/blob/master/xml/FrameworksIndex/healthvault-1.65.20623.2-preview.xml) | :white_check_mark:Succeeded | |
[xml/FrameworksIndex/healthvaultclient-1.65.20615.4-preview.xml](https://github.com/dend/healthvault-docs/blob/master/xml/FrameworksIndex/healthvaultclient-1.65.20615.4-preview.xml) | :white_check_mark:Succeeded | |
[xml/FrameworksIndex/healthvaultclient-1.65.20623.2-preview.xml](https://github.com/dend/healthvault-docs/blob/master/xml/FrameworksIndex/healthvaultclient-1.65.20623.2-preview.xml) | :white_check_mark:Succeeded | |
[xml/FrameworksIndex/healthvaultweb-1.65.20615.4-preview.xml](https://github.com/dend/healthvault-docs/blob/master/xml/FrameworksIndex/healthvaultweb-1.65.20615.4-preview.xml) | :white_check_mark:Succeeded | |
[xml/FrameworksIndex/healthvaultweb-1.65.20623.2-preview.xml](https://github.com/dend/healthvault-docs/blob/master/xml/FrameworksIndex/healthvaultweb-1.65.20623.2-preview.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.Client/HealthVaultConnectionFactory.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.Client/HealthVaultConnectionFactory.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.Clients/IHealthVaultRestClient.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.Clients/IHealthVaultRestClient.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.Clients/IPersonClient.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.Clients/IPersonClient.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.Clients/IThingClient.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.Clients/IThingClient.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.Clients/IVocabularyClient.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.Clients/IVocabularyClient.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.Configuration/HealthVaultConfiguration.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.Configuration/HealthVaultConfiguration.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.Connection/IHealthVaultConnection.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.Connection/IHealthVaultConnection.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.Connection/SessionCredential.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.Connection/SessionCredential.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.Exceptions/ConversionFailureException.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.Exceptions/ConversionFailureException.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.Exceptions/HealthHttpException.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.Exceptions/HealthHttpException.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.Exceptions/HealthRecordAuthorizationNotPossibleException.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.Exceptions/HealthRecordAuthorizationNotPossibleException.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.Exceptions/HealthRecordAuthorizationRequiredException.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.Exceptions/HealthRecordAuthorizationRequiredException.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.Exceptions/HealthRecordReauthorizationRequiredException.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.Exceptions/HealthRecordReauthorizationRequiredException.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.Exceptions/HealthServiceAccessDeniedException.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.Exceptions/HealthServiceAccessDeniedException.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.Exceptions/HealthServiceApplicationDuplicateCredentialException.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.Exceptions/HealthServiceApplicationDuplicateCredentialException.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.Exceptions/HealthServiceAuthenticatedSessionTokenExpiredException.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.Exceptions/HealthServiceAuthenticatedSessionTokenExpiredException.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.Exceptions/HealthServiceCredentialTokenExpiredException.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.Exceptions/HealthServiceCredentialTokenExpiredException.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.Exceptions/HealthServiceEmailNotValidatedException.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.Exceptions/HealthServiceEmailNotValidatedException.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.Exceptions/HealthServiceException.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.Exceptions/HealthServiceException.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.Exceptions/HealthServiceInvalidApplicationAuthorizationException.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.Exceptions/HealthServiceInvalidApplicationAuthorizationException.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.Exceptions/HealthServiceInvalidPersonException.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.Exceptions/HealthServiceInvalidPersonException.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.Exceptions/HealthServiceInvalidRecordException.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.Exceptions/HealthServiceInvalidRecordException.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.Exceptions/HealthServiceMailAddressMalformedException.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.Exceptions/HealthServiceMailAddressMalformedException.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.Exceptions/HealthServicePasswordNotStrongException.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.Exceptions/HealthServicePasswordNotStrongException.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.Exceptions/HealthServiceRecordQuotaExceededException.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.Exceptions/HealthServiceRecordQuotaExceededException.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.Exceptions/IncompatibleVersionException.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.Exceptions/IncompatibleVersionException.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.Exceptions/InvalidConfigurationException.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.Exceptions/InvalidConfigurationException.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.Exceptions/SignatureFailureException.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.Exceptions/SignatureFailureException.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.Exceptions/ThingDeserializationException.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.Exceptions/ThingDeserializationException.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.Exceptions/ThingSerializationException.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.Exceptions/ThingSerializationException.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.Exceptions/TypeHandlerAlreadyRegisteredException.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.Exceptions/TypeHandlerAlreadyRegisteredException.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.ItemTypes/ApplicationSpecific.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.ItemTypes/ApplicationSpecific.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.ItemTypes/ApproximateTime.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.ItemTypes/ApproximateTime.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.ItemTypes/Blob.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.ItemTypes/Blob.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.ItemTypes/HealthGoal.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.ItemTypes/HealthGoal.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.ItemTypes/LabTestResultDetails.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.ItemTypes/LabTestResultDetails.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.Person/PersonInfo.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.Person/PersonInfo.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.RestApi.Generated.Models/ActionPlan.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.RestApi.Generated.Models/ActionPlan.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.RestApi.Generated.Models/ActionPlanFrequencyTaskCompletionMetricsV2.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.RestApi.Generated.Models/ActionPlanFrequencyTaskCompletionMetricsV2.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.RestApi.Generated.Models/ActionPlanInstance.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.RestApi.Generated.Models/ActionPlanInstance.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.RestApi.Generated.Models/ActionPlanInstanceV2.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.RestApi.Generated.Models/ActionPlanInstanceV2.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.RestApi.Generated.Models/ActionPlanTaskInstance.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.RestApi.Generated.Models/ActionPlanTaskInstance.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.RestApi.Generated.Models/ActionPlanTaskInstanceV2.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.RestApi.Generated.Models/ActionPlanTaskInstanceV2.xml) | :white_check_mark:Succeeded | |
[xml/Microsoft.HealthVault.RestApi.Generated.Models/ActionPlanTaskV2.xml](https://github.com/dend/healthvault-docs/blob/master/xml/Microsoft.HealthVault.RestApi.Generated.Models/ActionPlanTaskV2.xml) | :white_check_mark:Succeeded | |

This comment lists only the first 50 files in the pull request.
For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C6%5Cd3a7b26e-e832-dfa2-6527-b27014cc5d28%5CPullRequest%5C201707062342279380-29%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: xml/Microsoft.HealthVault.ItemTypes/HealthGoal.xml [hunk 1]
```diff
@@ -225,7 +225,7 @@
              </summary>
         <value>To be added.</value>
         <remarks>
-             A goal can be defined on a weekly interval, meaning the target is intended to be achieved every week. Walking 70000 steps in a week is an example of this.
+             A goal can be defined on a weekly interval, meaning the target is intended to be achieved every week. Walking 70000 steps in a week is an example of
```
Comment (davidkwoods) [inline xml/Microsoft.HealthVault.ItemTypes/HealthGoal.xml]: Removal of " this." from comments was unintentional.  Can we manually add them back for now while we restore them in the source?

File: xml/Microsoft.HealthVault.ItemTypes/LabTestResultDetails.xml [hunk 2]
```diff
@@ -124,7 +124,7 @@
              Formatting:
              Notes may come from systems that require a line-based approach to textual display. To support
              this, applications should render the spaces and newlines in the note, and use a fixed-width font.
-             The form transform for the lab test results type shows one way to do this.
+             The form transform for the lab test results type shows one way to do
```
Comment (davidkwoods) [inline xml/Microsoft.HealthVault.ItemTypes/LabTestResultDetails.xml]: this.

File: xml/Microsoft.HealthVault.Thing/ThingKey.xml [hunk 3]
```diff
@@ -93,7 +93,7 @@
       </Parameters>
       <Docs>
         <param name="obj">
-             The <see cref="T:Microsoft.HealthVault.Thing.ThingKey" /> to compare against this.
+             The <see cref="T:Microsoft.HealthVault.Thing.ThingKey" /> to compare against
```
Comment (davidkwoods) [inline xml/Microsoft.HealthVault.Thing/ThingKey.xml]: this.

File: xml/Microsoft.HealthVault.ItemTypes/HealthGoal.xml (hunk 1 above)
Comment (dend) [inline xml/Microsoft.HealthVault.ItemTypes/HealthGoal.xml]: @davidkwoods yes, should be able to fix for now in this commit.

**Comment by dend** [comment]
@davidkwoods I will close this PR and open a new one with the updated version in tow.

**Pull request closed without merging**